guides/common/modules/proc_configuring-freeipa-authentication-on-server.adoc

```diff
@@ -63,7 +63,7 @@ endif::[]
 ----
 +
 Replace _OTP_ with the one-time password provided by the {FreeIPA} administrator.
-ifndef::satellite[]
+ifndef::satellite,orcharhino[]
```

Comment:
@evgeni should this be
```suggestion
ifdef::foreman-deb[]
```

Reply:
Sounds reasonable, and then we should drop the "If …" part - the same applies to Ubuntu too.